www/package.json

```diff
@@ -75,6 +75,7 @@
     "react-icons": "4.4.0",
     "react-json-view": "1.21.3",
     "react-portal": "4.2.2",
+    "react-resize-detector": "^7.1.2",
```

Comment:
I think it will be safer to stick to exact versions, without `~` or `^`.

Reply:
sure, will change

Reply:
You need to run yarn again as lock file still has old version inside.